src/gui/owncloudgui.cpp

```diff
@@ -85,6 +85,9 @@ ownCloudGui::ownCloudGui(Application *parent)
     connect(_tray.data(), &Systray::openHelp,
         this, &ownCloudGui::slotHelp);
 
+    connect(_tray.data(), &Systray::openMainDialog,
+        this, &ownCloudGui::slotOpenSettingsDialog);
```

Comment:
You meant `slotOpenMainDialog` here, didn't you? Otherwise it'd show the settings dialog when we click on "Open main dialog" I think.

Reply:
I know what I wanted with this: to check if the user was logged in or not. That is what slotOpenSettingsDialog does... and that is why I had added that code to slotShowSettings... I guess I should just display the main dialog.